lms/templates/courseware/course_about.html

```diff
@@ -40,9 +40,14 @@
           location.href = xhr.responseText;
         }
       } else if (xhr.status == 403) {
```

Comment:
The error text ("An error has occurred. Please ensure that you are logged in...") makes it feel like being not logged in is one of several error states for this, but it's our best guess. If I'm logged in already, could I still recieve a 403 in some edge case? Which would then redirect me to the signin, which I think would then redirect me to the dashboard. And the user wouldn't understand that we didn't actually enroll them.

Scanning through the enrollment API end point (which I believe is in `student/views/management.py` yeah?), it looks like it only does a 403 if unauthenticated. In which case, ignore my concern above. But can you also just double-check my scan?

Reply:
Yes, we found it only throws 403 only if unauthenticated in our check as well.